### ISSUE TITLE
Not working 

### ISSUE THREAD

**Issue by slowworks** [FreezyEx/PreSale-Contract#6]
Hey thanks for the presale contract,

I successful deployed it but when ever I send bnb to the contract address it fails and no error is emitted please can you explain what I did wrong 

My token details
decimal:- 8
hardcap:-  500000000000000000000 (500 bnb)
softcap:- 50000000000000000000 (50 bnb)
minbuy:- 1000000000000000 (0.001 bnb)
Maxbuy:- 1000000000000000000 (1 bnb)
rate:- 100000000000 (1000 token = 1 bnb)

**Comment by FreezyEx** [comment]
did you set gas limit to 300k?

Il Gio 5 Ago 2021, 00:15 Slow Works ***@***.***> ha scritto:

**Comment by slowworks** [comment]
Yes @FreezyEx  I did but transaction still failed I think I did a mistake but I have no idea what and hey my token is 8 decimal will I still use Wei for all fields or I use gwei??
